pipelines/folioRancher/folioTestingTools/folioSchemaCompare.groovy

```diff
@@ -0,0 +1,221 @@
+#!groovy
+import hudson.util.Secret
+import org.jenkinsci.plugins.workflow.libs.Library
+import java.time.*
+import org.folio.rest.GitHubUtility
+import org.folio.Constants
+import groovy.json.JsonSlurperClassic
+
+@Library('pipelines-shared-library@RANCHER-838') _
+
+String getOkapiVersion(folio_repository, folio_branch) {
+    def installJson = new URL('https://raw.githubusercontent.com/folio-org/' + folio_repository + '/' + folio_branch + '/install.json').openConnection()
+    if (installJson.getResponseCode().equals(200)) {
+        String okapi = new JsonSlurperClassic().parseText(installJson.getInputStream().getText())*.id.find{it ==~ /okapi-.*/}
+        if(okapi){
+            return okapi - 'okapi-'
+        } else {
+            error("Can't get okapi version from install.json in ${folio_branch} branch of ${folio_repository} repository!" )
+        }
+    }
+    error("There is no install.json in ${folio_branch} branch of ${folio_repository} repository!" )
+}
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '20')),
+    disableConcurrentBuilds(),
+    parameters([
+        jobsParameters.refreshParameters(),
+        jobsParameters.repository(),
+        jobsParameters.branch('folio_repository', 'folio_branch_src'),
+        jobsParameters.branch('folio_repository', 'folio_branch_dst'),
+        string(name: 'backup_name', defaultValue: '', description: '(Optional) RDS snapshot name. If empty create env from scratch', trim: true),
+        string(name: 'slackChannel', defaultValue: '', description: 'Slack channel name where send report (without #)', trim: true)])])
+
+def rancher_cluster_name = 'folio-perf'
```

Comment:
Please check variables naming example in pipelines/folioRancher/folioNamespaceTools/manageNamespace/createNamespaceFromBranch/Jenkinsfile

we switching from snake_case to camelCase

Reply:
Dear Oleksandr
cameCase to change , than entire trigger job also should be change
because triggered jobs are under snake case
and what you think, if you give me permission
for this styling we can have separate small ticket to redesign
to save time for this job
and will do definitely